src/pal/src/loader/module.cpp

```diff
@@ -676,7 +676,7 @@ PAL_RegisterModule(
 {
     HINSTANCE hinstance = nullptr;
 
-    int err = PAL_InitializeDLL();
+    int err = PAL_InitializeDLLWithoutSEH();
```

Comment:
That isn’t exactly true that the debugger dlls don’t use LoadLibrary.  Yes, the libsosplugin.so uses dlopen to load libsos.so, but libsos uses LoadLibrary to load libmscordaccore.so and libmscordbi.so.  And libmscordbi.so uses LoadLibrary to load libmscordaccore too.

But they all could be changed not to depend on there DLLMain handlers being called and explicitly calling PAL_InitializeDLL (I think libsos already does).

Now VS may have a problem with a DLLMain not being called. @gregg-miskelly what do you think?